action.yaml

```diff
@@ -68,10 +68,32 @@ inputs:
     description: 'Force purge and deletion of tf_state_bucket defined. Any file contained there will be destroyed. tf_stack_destroy must also be true.'
     default: "false"
 
+  # Domains
+  domain_name:
+    description: 'Define the root domain name for the application. e.g. app.com'
+    required: false
+  sub_domain:
+    description: 'Define the sub-domain part of the URL. Defaults to `${org}-${repo}-{branch}`'
+  root_domain:
+    description: 'Deploy to root domain. Will generate two DNS recrods, one for root, another for www'
+    required: false
+  cert_arn:
+    description: 'Define the certificate ARN to use for the application'
+    required: false
+  create_root_cert:
+    description: 'Generates and manage the root cert for the application'
+    required: false
+  create_sub_cert: 
+    description: 'Generates and manage the sub-domain certificate for the application'
+    required: false
```

Comment:
Not really. If the user want's to create and manage a root cert, it's possible to do it here.
If just a sub-domain is wanted, that can be done here too. 
The user can set up a sub_domain.domain_name with a root cert and it will work, and so with create sub_cert. 

We could change the behavior though.